Versions/Common/BeStride_Constants.lua

```diff
@@ -173,9 +173,6 @@ BeStride_Constants = {
 				Worlds = {
 				},
 				Continents = {
-					[113] = {
-						requires = 54197,
-					},
```

Comment:
Cold Weather Flying apparently hasn't been required on retail since [7.3.5](https://www.wowhead.com/spell=54197/cold-weather-flying#comments:id=2538905)

The Wrath version adds 54197 back in via a new OverrideConstants function that runs on init.